vars.tf

```diff
@@ -1,23 +1,39 @@
-variable "api_gateway_rest_api_name" {}
+variable "api_gateway_rest_api_name" {
+  description = "Name of the REST API on API gateway"
+}
 
 variable "api_gateway_endpoint_method" {
-  default = "GET"
+  default     = "GET"
+  description = "HTTP method for the API gateway proxy endpoint"
 }
 
 variable "api_gateway_deployment_stage_name" {
-  default = "development"
+  default     = "development"
+  description = "API gateway stage to publish endpoint on"
+}
+
+variable "lambda_function_name" {
+  description = "Name of the lambda on AWS"
+}
+
+variable "lambda_handler" {
+  description = "Name of the lambda handler module"
 }
 
-variable "lambda_function_name" {}
-variable "lambda_handler" {}
 variable "lambda_handler_method" {
-  default = "handler"
+  default     = "handler"
+  description = "Name of the lambda handler method"
 }
+
 variable "lambda_runtime" {
-  default = "nodejs8.10"
+  default     = "nodejs8.10"
+  description = "Node runtime for the lambda on AWS"
```

Comment:
[nit] This wouldn't necessarily need to be nodejs. We could start writing these in Go (or any other supported runtime).